content/en/blog/2022/gc-elections-2022.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Announcing the 2022 OpenTelemetry Governance Committee Election
+linkTitle: OpenTelemetry Governance Committee Election 2022
+date: 2022-09-14
```

Comment:
@jpkrohling , let's update the date to 2022-09-15 and then we can merge & publish!